Builder/include/CodeBuilder.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <ostream>
+#include <sstream>
+#include <iostream>
+
+#ifndef CODESNIPPET_H
+#define CODESNIPPET_H
+class CodeSnippet
+{
+	std::string class_name;
```

Comment:
It's more readable if you place the private members after the public ones and also use the private keyword.